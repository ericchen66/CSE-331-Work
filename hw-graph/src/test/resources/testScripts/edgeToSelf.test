# Tests creating an edge from a node to the same node

# Creates a graph
CreateGraph graph1

# Adds a node to the graph
AddNode graph1 node1

# Creates an edge from the node to itself
AddEdge graph1 node1 node1 edge1

# Lists the child node of the node
ListChildren graph1 node1