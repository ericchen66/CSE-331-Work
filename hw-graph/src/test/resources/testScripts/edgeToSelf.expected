# Tests creating an edge from a node to the same node

# Creates a graph
created graph graph1

# Adds a node to the graph
added node node1 to graph1

# Creates an edge from the node to itself
added edge edge1 from node1 to node1 in graph1

# Lists the child node of the node
the children of node1 in graph1 are: node1(edge1)